docs/features/servicebus-queue-worker-template.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "ServiceBus Queue worker template"
+layout: default
+---
+
+# ServiceBus Queue Worker Project Template
+
+## Create Your First Arcus ServiceBus Queue Worker Project
+
+First, install the template from NuGet:
+
+```shell
+> dotnet new --install Arcus.Templates.ServiceBus.Queue
+```
+
+When installed, the template can be created with shortname: `arcus-servicebus-queue`:
+
+```shell
+> dotnet new arcus-servicebus-queue --name Arcus.Demo.ServiceBus.Queue 
+```
+
+
+## Features
+
+Creates a starter worker project with by default configured:
```

Comment:
Should we point to official docs, our docs on the features or both?

Reply:
Don't get what you mean. Now  we have two references to the official docs for the two available features. What is your concern?

Reply:
I'm suggesting to point to our docs as well, for example:
* TCP health check probe. ([Arcus docs](https://messaging.arcus-azure.net/features/tcp-health-probe) & [official docs](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-2.2))

Just a suggestion though but then the know how to tweak those and what is part of the package.

Reply:
Ah, ok; yeah makes sense.

Reply:
Updated.